linux_os/guide/services/fapolicyd/configure_fapolicyd_mounts/rule.yml

```diff
@@ -0,0 +1,31 @@
+documentation_complete: true
+
+prodtype: rhel8,fedora,ol8
+
+title: 'Enable fapolicyd to watch all system mountpoints'
+
+description: |-
+    Configure the <tt>fapolicyd</tt> (File Access Policy Daemon)
+    to watch on all system filesystem mountpoints
+    in <tt>/etc/fapolicyd/fapolicyd.mounts</tt>.
+
+rationale: |-
+    To control file access for all possible rules, all system
```

Comment:
Do you mean  `all possible mountpoints`?

Reply:
I think I had rules in mind -- to support any paths you might want to put to the `.rules` file that may land on any filesystem, we need to list all filesystems.

Reply:
Not sure I understand correctly.
So `fapolicyd.mounts` lists partitions to watch, and paths in `fapolicyd.rules` should fall within watched mounts?
Otherwise the rule won't have any effect?

Reply:
That's my understanding of the rather sparse `fapolicyd(8)` man page.